tests/Unit/ConfigurationTest.php

```diff
@@ -100,6 +100,26 @@ public function testAppNameFallbackPriorities()
         $this->assertSame('bar_app', Configuration::get()->appName());
     }
 
+    public function testServiceName()
+    {
+        Configuration::clear();
```

Comment:
I think we need to reset the fallback env var before this test so that other test env vars don't leak into this one. :)

```suggestion
        putenv('DD_TRACE_APP_NAME');
        Configuration::clear();
```

Reply:
Good catch 👍

Reply:
@SammyK does the change ^^ work for you?

Reply:
@labbati Perrrrrfect! 🐈